File: solutions/problem_0063/java/Main.java
/*
 * Elaborado por: Ednilson G Rossi
 * linktr.ee/ednilsonrossi
 */

package Project_PS63.src;


/**
 * Main
 */
public class Main {

    private static final int TERMO_DESEJADO = 20;

    public static void main(String[] args) {
        int termoA, termoB, termoC;
        int sequencia;

        termoA = 0;
        termoB = 1;
        System.out.println("1º termo: " + termoA);
        System.out.println("2º termo: " + termoB);

        sequencia = 3;
        while (sequencia <= TERMO_DESEJADO) {
            termoC = termoA + termoB;
            System.out.printf("%dº termo: %d\n", sequencia, termoC);

            termoA = termoB;
            termoB = termoC;
            sequencia++;
        }
    }
}

File: solutions/problem_0063/kotlin/Main.kt
fun main() {
    var termoA: Int
    var termoB: Int
    var termoC: Int
    var sequencia: Int

    termoA = 0;
    termoB = 1;

    println("1º termo: $termoA")
    println("2º termo: $termoB")

    sequencia = 3
    while (sequencia <= 20){
        termoC = termoA + termoB
        println("${sequencia}º termo: $termoC")

        termoA = termoB
        termoB = termoC
        sequencia += 1
    }
}